app/javascript/courses/review/components/CoursesReview__Editor.res

```diff
@@ -1184,6 +1210,7 @@ let make = (
                 <div className="py-4 md:ml-8 text-center">
                   <button
                     onClick={_ => send(ShowAdditionalFeedbackEditor)}
+                    disabled={SubmissionDetails.preview(submissionDetails)}
```

Comment:
I think two more elements in this page should be disabled for _pending_ submissions in preview mode: the review checklist `<button>`, and the feedback Markdown editor.